.github/workflows/build.yaml

```
@@ -60,5 +68,5 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           context: ./${{ matrix.directory }}
-          push: ${{ env.PUSH_IMAGE }}
+          push: ${{ fromJSON(steps.eval_push.outputs.PUSH_IMAGE) }}
```

This can probably be an inline function:

```suggestion
          push: ${{ github.event_name == 'push' && true || false }}
```